src/ast/value.rs

```diff
@@ -182,8 +182,27 @@ pub struct EscapeQuotedString<'a> {
     quote: char,
 }
 
+
 impl<'a> fmt::Display for EscapeQuotedString<'a> {
     fn fmt(&self, f: &mut fmt::Formatter) -> fmt::Result {
+        // EscapeQuotedString doesn't know which mode of escape was
```

Comment:
This is mostly a reformatted version of @canalun 's comments in https://github.com/sqlparser-rs/sqlparser-rs/pull/870/files#r1240843867